appserver/distributions/web/pom.xml

```diff
@@ -71,6 +71,11 @@
             <scope>test</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
We already abandoned Junit4 ;-)

Reply:
However this is glassfish-distribution packaging and even when you compile those classes, they will not be executed because surefire/failsafe plugins are not in its lifecycle ... or am I wrong?
Still would be better to migrate these tests tu JUnit5 than poisoning build with the obsoleted version.

Reply:
I rechecked - the failsafe plugin is excuted, so just rename the test class to *IT,m use junit5-jupiter-engine as a dependency and use @BeforeAll + @AfterAll etc. from JUnit5. But really don't bring Junit4 back to the project, pleeeease! ;-)
```
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-engine</artifactId>
        </dependency>

```

Reply:
> But really don't bring Junit4 back to the project, pleeeease! ;-)

It was essentially already there :P